### ISSUE TITLE
Add "skip to main" link

### ISSUE THREAD

**Issue by YuriDevAT** [YurisCodingClub/dev-cv#31]
### Description

Add a skip to main link to the project by following the [instructions of WebAIM](https://webaim.org/techniques/skipnav/) on how to do so.

## Acceptance criteria (AC)

- [ ] content (all the form components) is wrapped into a `main` tag
- [ ] the skip link should be `invisible` and only appears when using the `tab` key

### Additional information

Suggestion on  how to hide the skip link and make it visible when receiving focus. Should be tested.

```
.skip-link {
  left: 0;
  padding: 8px;
  position: absolute;
  top: -50px;
  z-index: 100;
}

.skip-link:focus {
  top: 0;
}
```

**Comment by amardeep911** [comment]
Plz assign this to me

**Comment by jsvigneshkanna** [comment]
@YuriDevAT , assign me this issue

**Comment by siddhesh-coder** [comment]
@YuriDevAT assign me this issue

**Comment by hariscs** [comment]
hey @YuriDevAT PR for this issue is ready just review it [here](https://github.com/YurisCodingClub/dev-cv/pull/39)
